crates/bevy_pbr/src/render/mesh_preprocess_types.wgsl

```diff
@@ -2,18 +2,96 @@
 
 #define_import_path bevy_pbr::mesh_preprocess_types
 
-// The `wgpu` indirect parameters structure. This is a union of two structures.
-// For more information, see the corresponding comment in
-// `gpu_preprocessing.rs`.
-struct IndirectParameters {
-    // `vertex_count` or `index_count`.
-    vertex_count_or_index_count: u32,
-    // `instance_count` in both structures.
-    instance_count: atomic<u32>,
-    // `first_vertex` or `first_index`.
-    first_vertex_or_first_index: u32,
-    // `base_vertex` or `first_instance`.
-    base_vertex_or_first_instance: u32,
-    // A read-only copy of `instance_index`.
+// Per-frame data that the CPU supplies to the GPU.
+struct MeshInput {
+    // The model transform.
+    world_from_local: mat3x4<f32>,
+    // The lightmap UV rect, packed into 64 bits.
+    lightmap_uv_rect: vec2<u32>,
+    // Various flags.
```

Comment:
I know this comment is just moved from another file, but this comment sucks. What flags are these? You don't have to resolve this, it can be dont in another (potentially someone else's) PR.

Reply:
I updated the comment to mention that these are `MeshFlags`.